.gitignore

```diff
@@ -1 +1,2 @@
 /target
+**/.DS_Store
```

Comment:
@appcypher Protip: you can add this to your system gitignore, and never have to deal with this in projects again! https://www.flother.is/til/global-gitignore/

Reply:
Handy